docs/inference-providers/providers/featherless-ai.md

```diff
@@ -19,6 +19,10 @@ For more details, check out the `generate.ts` script: https://github.com/hugging
 
 # Featherless AI
 
+<Tip>
+All supported  models can be found [here](https://huggingface.co/models?inference_provider=&sort=trending)
```

Comment:
This results in a 404, I assume it needs to be https://huggingface.co/models?inference_provider=featherless-ai&sort=trending

Reply:
hmm.. let me see what went wrong